web-client/public/js/generateNetwork.js

```diff
@@ -242,6 +248,7 @@ containing "-", "_", and alpha-numeric characters only`);
  ${genesAmount} genes. Please remove some genes from your proposed network.`);
         } else {
             if (grnState.customWorkbook.type === NETWORK_GRN_MODE) {
+                grnState.customWorkbook.genes = sortGenes(grnState.customWorkbook.genes, 1);
```

Comment:
That way, over here, we have named constants rather than hardcoded numbers:
```suggestion
                grnState.customWorkbook.genes = sortGenes(grnState.customWorkbook.genes, GRN_STANDARD_NAME_ID);
```